src/app/examples/modal-example/second-embedded-modal-example/second-embedded-modal-example.component.tns.html

```diff
@@ -0,0 +1,4 @@
+<StackLayout> 
+    <Label textWrap="true" text="Hello from the second embedded example component"></Label>
+    <Button kirby-button text="Hide me" (tap)="onHidePress($event)"></Button>
```

Comment:
We use the term `select` to cover both `click` and `tap`event across web + {N}:
```suggestion
    <Button kirby-button text="Hide me" (tap)="onHideSelect($event)"></Button>
```

OR simply:
```suggestion
    <Button kirby-button text="Hide me" (tap)="hideModal($event)"></Button>
```